connect/src/com/telenor/connect/id/ConnectIdService.java

```diff
@@ -182,13 +192,13 @@ private void clearCookies(Context context) {
         }
     }
 
-    public void logOut(final Context context) {
+    public void logOut(final Context context, final boolean shouldNotify) {
```

Comment:
It seems like this method is always called with shouldNotify=true, so maybe adding the parameter is not needed, neither for logOut nor for revokeTokens?

Reply:
Yep, that's left from revoke testing, will be fixed.